packages/space-opera/src/components/utils/render_model_viewer.ts

```diff
@@ -34,6 +39,51 @@ export interface ModelViewerEventHandlers {
   readonly error?: (details: CustomEvent) => void;
 }
 
+export async function createPoster() {
+  const modelViewer = getModelViewer();
+  const ModelViewerElement = customElements.get('model-viewer');
+  const oldMinScale = ModelViewerElement.minimumRenderScale;
+  ModelViewerElement.minimumRenderScale = 1;
+
+  const state = reduxStore.getState();
+  const poster = getPosterConfig(state);
+
+  const height = poster.height / window.devicePixelRatio;
+  modelViewer.style.width = `${height}px`;
+  modelViewer.style.height = `${height}px`;
+
+  // Set to beginning of animation
+  const oldTime = modelViewer.currentTime;
+  modelViewer.autoplay = false;
+  modelViewer.currentTime = 0;
```

Comment:
would it make sense to have reset or init function for this part?

Reply:
Maybe, but since the reset is pretty specific to the poster generation I feel like it's simpler to contain the logic here. If it comes in hand anywhere else, I'll  definitely put it in a function.